app/presenters/licences/set-up.presenter.js

```diff
@@ -161,6 +170,21 @@ function _financialAgreementCode (agreement) {
   return agreement.financialAgreements[0].financialAgreementCode
 }
 
+function _returnsVersions (returnVersions = [{}]) {
```

Comment:
NIT - we call the property we're setting `returnVersion` and we pass in `returnVersions`. Sorry, should have spotted on first pass.

```suggestion
function _returnVersions (returnVersions = [{}]) {
```